src/utils.ts

```diff
@@ -7,12 +7,36 @@ import { Contract } from "ethers";
 import { FORKED_NETWORK, config, ethers, network } from "hardhat";
 
 import { NETWORK_ADDRESSES } from "./networkAddresses";
-import { Command, Proposal, ProposalMeta, ProposalType, TokenConfig } from "./types";
+import { Command, LzChainId, Proposal, ProposalMeta, ProposalType, REMOTE_NETWORKS, TokenConfig } from "./types";
+import OmnichainProposalSender_ABI from "./vip-framework/abi/OmnichainProposalSender_ABI.json";
 import VENUS_CHAINLINK_ORACLE_ABI from "./vip-framework/abi/VenusChainlinkOracle.json";
 import BINANCE_ORACLE_ABI from "./vip-framework/abi/binanceOracle.json";
 import CHAINLINK_ORACLE_ABI from "./vip-framework/abi/chainlinkOracle.json";
 import COMPTROLLER_ABI from "./vip-framework/abi/comptroller.json";
 
+const testnetNetworks = ["sepolia", "opbnbtestnet", "arbitrumsepolia"];
+const mainnetNetworks = ["ethereum", "opbnbmainnet", "arbitrumone"];
```

Comment:
Could we reuse or merge these lists with the types defined in the `types.ts` file (like `REMOTE_NETWORKS`)?

Reply:
It would simplify the process of adding a new network